azext_iot/central/_help.py

```diff
@@ -276,30 +278,31 @@ def _load_central_api_token_help():
         "iot central api-token"
     ] = """
         type: group
-        short-summary: Create and Manage API tokens.
+        short-summary: Manage API tokens for your IoT Central application.
+        long-summary: IoT Central allows you to generate and manage API tokens to be used to access the IoT Central API. More information about APIs can be found at https://aka.ms/iotcentraldocsapi.
     """
 
     helps[
         "iot central api-token create"
     ] = """
         type: command
-        short-summary: Create a new API token in the application
-        long-summary: The only time you will see the value of this token is when creating the token. Ensure you store this token somewhere securely, as if you lose it, you will need to create another.
+        short-summary: Generate an API token associated with your IoT Central application.
+        long-summary: |
+            Note: Write down your token once it's been generated as you won't be able to retrieve it again.
         examples:
         - name: Add new API token
           text: >
             az iot central api-token create
             --token-id {tokenId}
             --app-id {appId}
             --role admin
-
     """
     helps[
         "iot central api-token show"
     ] = """
     type: command
-    short-summary: Get token meta data (e.g. role as a GUID, expiration)
-    long-summary: API token information contains basic information about the token and does not include the value of the token.
+    short-summary: Get details for an API token associated with your IoT Central application.
+    long-summary: List details, like its associated role, for an API token in your IoT Central app.
```

Comment:
e.g. this long-summary is not multi-line so we dont need the `: |` since that indicates multi line. We should do something similar for api token